core/src/main/java/hudson/model/ItemGroupMixIn.java

```diff
@@ -104,11 +104,11 @@ public boolean accept(File child) {
                 // Try to retain the identity of an existing child object if we can.
                 V item = (V) parent.getItem(subdir.getName());
                 if (item == null) {
-                    XmlFile xmlFile = Items.getConfigFile( subdir );
+                    XmlFile xmlFile = Items.getConfigFile(subdir);
```

Comment:
Please avoid whitespace-only changes in PRs. This whole diff hunk appears to be gratuitous.